src/GraphQL.Client/GraphQLHttpClient.cs

```diff
@@ -196,7 +188,7 @@ protected virtual void Dispose(bool disposing)
                 Debug.WriteLine($"Disposing GraphQLHttpClient on endpoint {Options.EndPoint}");
                 _cancellationTokenSource.Cancel();
                 HttpClient.Dispose();
-                _graphQlHttpWebSocket?.Dispose();
+                _lazyHttpWebSocket?.Value.Dispose();
```

Comment:
```suggestion
                if(_lazyHttpWebSocket?.HasValue) _lazyHttpWebSocket?.Value.Dispose();
```

Otherwise we'll create an instance on calling `Dispose()` 😉